facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -3,79 +3,112 @@
 @import layout.EditionalisedLink
```

Comment:
The imports in this file are a bit of a mess. I would move ` @(page: model.PressedPage)(implicit request: RequestHeader)` on line 9 to the top, and IntelliJ can help you clean up the imports.